ts/Maps/MapView.ts

```diff
@@ -415,7 +415,13 @@ class MapView {
 
     public getGeoMap(map?: MapDataType): GeoJSON|undefined {
         if (isString(map)) {
-            return maps[map];
+            const mapFromArray = maps[map];
+
+            if (mapFromArray && mapFromArray.type === 'Topology') {
+                return topo2geo(mapFromArray);
+            }
```

Comment:
Do we need to create an extra variable for `maps[map]`? In my opinion, we don't need that declaration.